<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="6d85e861-f102-4641-a54c-2f3b941967f6">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\PyChain\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="78b2e1d7-832c-4352-b0f3-570faf4d6016">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/PyChain/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:D:\PyChain\db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>